src/Core/Containers/Grid.hpp

```diff
@@ -206,8 +206,8 @@ class Grid {
     inline T& at( uint idx );
 
     /// Access an element with an iterator.
-    inline const T& at( const Iterator& it ) const;
-    inline T& at( const Iterator& it );
+//    inline const T& at( const Iterator& it ) const;
+//    inline T& at( const Iterator& it );

```

Comment:
If these methods are no longer used, remove them instead of just commenting them.

Reply:
restored